sender.py

```diff
@@ -19,9 +19,9 @@ def send(file, ip):
             click.echo(f'Encoder with serial number {SENDER_SERIAL_NUMBER} has been successfully registered.')
     
     # This simply runs the following command
-    # ffmpeg -i <input> -f mpegts -v warning -stats udp://<ip>:23000
-    stream = ffmpeg.input(file)
-    stream = ffmpeg.output(stream, f'udp://{ip}:23000', f = 'mpegts', v = 'warning', stats = None)
+    # ffmpeg -re -i <input> -f mpegts -v warning -stats srt://<ip>:23000?pkt_size=1316
+    stream = ffmpeg.input(file, re = None)
```

Comment:
In the ffmpeg options I see this options is to "Read input at native frame rate. " [link](https://ffmpeg.org/ffmpeg.html#toc-Advanced-options). I'm curious why we are passing None to it here?

Reply:
Good question - so with ffmpeg-python (the python wrapper around ffmpeg) whenever we want to pass an option that doesn't take an argument (such as -re, as opposed to -f for example) we have to include that option and set its value to None. It's a bit clunky but that's the way it works.